crawl4ai/processors/pdf/__init__.py

```
@@ -3,80 +3,87 @@
-    def __init__(self, 
-                 save_images_locally : bool = False,
-                 extract_images : bool = False,
-                 image_save_dir : str = None,
-                 batch_size: int = 4,
-                 logger: AsyncLogger = None):
+
+    def __init__(
+        self,
+        save_images_locally: bool = False,
+        extract_images: bool = False,
+        image_save_dir: str = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)